src/constants/selectOptions.js

```diff
@@ -95,14 +95,17 @@ export const identifierOptions = {
     {
       value: IDENTIFIERS.FORMER_IDS,
       label: 'ui-bulk-edit.list.filters.recordIdentifier.item.former',
+      isCentralTenant: true,
     },
     {
       value: IDENTIFIERS.ACCESSION_NUMBER,
       label: 'ui-bulk-edit.list.filters.recordIdentifier.item.accession',
+      isCentralTenant: true,
     },
     {
       value: IDENTIFIERS.HOLDINGS_RECORD_ID,
       label: 'ui-bulk-edit.list.filters.recordIdentifier.item.holdingsUUID',
+      isCentralTenant: true,
     },
   ],
   [CAPABILITIES.HOLDING]: [
```

Comment:
`value` and `label` are self-explanatory in the context of an options-list but it's not obvious what `isCentralTenant` means. Include a comment somewhere that explains the same thing the story does, i.e. that only certain identifiers-options are visible for the central tenant? 

Also, given these options will _hidden_ in ECS envs for the central tenant, `isCentralTenant` is an awkward label. It reads like these options will only be displayed for the central tenant when the opposite is true. Consider sth like `isIgnoredInCentralTenant`?

Reply:
@zburke, thanks for that suggestion. I renamed it to `isIgnoredInCentralTenant`, so it's more clear to understand what is happening.